src/portal_to_terra_input_from_anaset.py

```diff
@@ -0,0 +1,298 @@
+import urllib.parse
+from igvf_and_terra_api_tools import get_igvf_auth_and_api
```

Comment:
igvf_and_terra_api_tools is not a great module name.  could just call it pipeline_tools.  In general I think you probalby want to separate API wrapper functions from scripts.